feconf.py

```diff
@@ -1092,7 +1092,8 @@ def get_empty_ratings():
 CLASSROOM_PAGES = ['math']
 
 # Authentication method using GAE ID (google sign in).
-AUTH_METHOD_GAE = 'gae'
+GAE_AUTH_PROVIDER_ID = 'gae'
+FIREBASE_AUTH_PROVIDER_ID = 'Firebase'
```

Comment:
Why capitalize one and not the other?

I lean a bit towards 'firebase' FWIW, it follows other naming conventions (we don't typically use UpperCamelCase).

Reply:
Resolved offline, this is OK.